alf/algorithms/sac_algorithm.py

```diff
@@ -203,6 +208,14 @@ def __init__(self,
                 Breakout. Only used for evaluation. If None, its value is taken
                 from ``config.epsilon_greedy`` and then
                 ``alf.get_config_value(TrainerConfig.epsilon_greedy)``.
+            rollout_epsilon_greedy (float): epsilon greedy policy for rollout.
+                Together with the following three parameters, the Sac algorithm
+                can be converted to a DQN algorithm.
+            use_epsilon_schedule (float): training schedule for
+                rollout_epsilon_greedy.
+            max_target_action (bool): whether to use the action with the highest
+                target value as the target action for computing bootstrapped value.
+                To mimic the DQN algorithm, set this to True.
```

Comment:
Feels a little strange to include these arguments here because there is usually no good reason to convert SAC to DQN. Maybe create a subclass of SAC for this purpose?

Reply:
> Yes, does feel hacky. The code is quite intertwined, so a subclass probably wouldn't do to separate the two classes neatly. Do you prefer to just remove this part for now?

I think it's definitely possible and clearer to have a subclass? You only need to include these additional arguments and their code in the subclass implementation.